monkestation/code/modules/antagonists/borers/code/antagonist_stuff/midround_event.dm

```diff
@@ -0,0 +1,111 @@
+/// How many people do we need per borer spawned
+#define POP_PER_BORER 30
+
+/datum/round_event_control/antagonist/solo/from_ghosts/cortical_borer
+	name = "Cortical Borer Infestation"
+	tags = list(TAG_TEAM_ANTAG, TAG_EXTERNAL, TAG_ALIEN)
+	typepath = /datum/round_event/ghost_role/cortical_borer
+	antag_flag = ROLE_BORER
+	enemy_roles = list(
+		JOB_CAPTAIN,
+		JOB_DETECTIVE,
+		JOB_HEAD_OF_SECURITY,
+		JOB_SECURITY_OFFICER,
+	)
+	required_enemies = 2
+	weight = 5 // as rare as a natural blob
+	min_players = 20
+	max_occurrences = 1 //should only ever happen once
+	dynamic_should_hijack = TRUE
+	category = EVENT_CATEGORY_ENTITIES
+	description = "A cortical borer has appeared on station. It will also attempt to produce eggs, and will attempt to gather willing hosts and learn chemicals through the blood."
+
+/datum/round_event/ghost_role/cortical_borer
+	announce_when = 400
+
+/datum/round_event/ghost_role/cortical_borer/setup()
+	announce_when = rand(announce_when, announce_when + 50)
+
+/datum/round_event/ghost_role/cortical_borer/announce(fake)
+	priority_announce(
+		"Unidentified lifesigns detected coming aboard [station_name()]. Secure any exterior access, including ducting and ventilation.",
+		"Lifesign Alert",
+		ANNOUNCER_ALIENS,
+	)
+
+/datum/round_event/ghost_role/cortical_borer/start()
+	var/list/vents = list()
+	for(var/obj/machinery/atmospherics/components/unary/vent_pump/temp_vent as anything in SSmachines.get_machines_by_type_and_subtypes(/obj/machinery/atmospherics/components/unary/vent_pump))
+		if(QDELETED(temp_vent))
+			continue
+		if(is_station_level(temp_vent.loc.z) && !temp_vent.welded)
+			var/datum/pipeline/temp_vent_parent = temp_vent.parents[1]
+			if(!temp_vent_parent)
+				continue // No parent vent
+			// Stops Cortical Borers getting stuck in small networks.
+			// See: Security, Virology
+			if(length(temp_vent_parent.other_atmos_machines) > 20)
+				vents += temp_vent
+	if(!length(vents))
+		return MAP_ERROR
+	var/list/mob/dead/observer/candidates = poll_ghost_candidates("Do you want to spawn as a cortical borer?", ROLE_PAI, FALSE, 10 SECONDS, POLL_IGNORE_CORTICAL_BORER)
+	if(!length(candidates))
+		return NOT_ENOUGH_PLAYERS
+	var/living_number = max(length(GLOB.player_list) / POP_PER_BORER, 1)
+	var/choosing_number = min(length(candidates), living_number)
+	for(var/repeating_code in 1 to choosing_number)
+		var/mob/dead/observer/new_borer = pick(candidates)
+		candidates -= new_borer
+		var/turf/vent_turf = get_turf(pick(vents))
+		var/mob/living/basic/cortical_borer/spawned_cb = new /mob/living/basic/cortical_borer(vent_turf)
+		spawned_cb.ckey = new_borer.ckey
+		spawned_cb.mind.add_antag_datum(/datum/antagonist/cortical_borer)
+		to_chat(spawned_cb, span_warning("You are a cortical borer! You can fear someone to make them stop moving, but make sure to inhabit them! You only grow/heal/talk when inside a host!"))
+	for(var/mob/dead_mob in GLOB.dead_mob_list)
```

Comment:
replace this with announce_to_ghosts(mobs)

Reply:
assuming ya mean to put announce_to_ghosts(spawned_cb) in the code above, le done?